master/internal/sproto/jobs.go

```diff
@@ -57,12 +57,6 @@ func DeleteJobResponseOf(input error) DeleteJobResponse {
 	return DeleteJobResponse{Err: respC}
 }
 
-// GetJobQStats requests stats for a queue.
-// Expected response: jobv1.QueueStats.
-type GetJobQStats struct {
-	ResourcePool string
-}
-
```

Comment:
Just double-checking this change wasn't included by accident? It's fine to include but seems slightly unrelated to the rest of the changes?

Reply:
Ah I'll add it in the commentary, but this is dead code -- I just came across it in this PR, but we should remove it anyway

Reply:
No need to include it in the commentary! It's fine as is.